ROADMAP.md

```diff
@@ -0,0 +1,185 @@
+
+| Status      | Target Completion | Tracking                                          | Dependencies                                                        | Dependents |
+|-------------|-------------------|---------------------------------------------------|---------------------------------------------------------------------|------------|
+| In progress | Q4/2022           | https://github.com/libp2p/rust-libp2p/issues/2883 | [TLS](#tls-support), https://github.com/libp2p/test-plans/issues/53 |            |
+
+QUIC has been on the roadmap for a long time. It enables various performance improvements as well as
+higher hole punching success rates. We are close to finishing a first version with
+https://github.com/libp2p/rust-libp2p/pull/2289.
+
+Long term there is lots more to do, see tracking issue
+https://github.com/libp2p/rust-libp2p/issues/2883. Though this will be tackled as a new roadmap
+item.
+
+## WebRTC support (browser-to-server)
+
+| Status      | Target Completion | Tracking                                 | Dependencies                                   | Dependents |
+|-------------|-------------------|------------------------------------------|------------------------------------------------|------------|
+| In progress | Q4/2022           | https://github.com/libp2p/specs/pull/412 | https://github.com/libp2p/test-plans/issues/53 |            |
+
+
+We are currently implementing WebRTC for **browser-to-server** connectivity in
+https://github.com/libp2p/rust-libp2p/pull/2622. More specifically the server side. This will enable
+browser nodes to connect to rust-libp2p nodes where the latter only have self-signed TLS
+certificates. See https://github.com/libp2p/specs/pull/412 for in-depth motivation.
+
+Long term we should enable rust-libp2p running in the browser via WASM to use the browser's WebRTC
+stack. Though that should only happen after improved WASM support, see below.
+
+## Cross Behaviour communication
+
+| Status | Target Completion | Tracking                                          | Dependencies                                      | Dependents                                    |
+|--------|-------------------|---------------------------------------------------|---------------------------------------------------|-----------------------------------------------|
+| todo   | Q1/2023           | https://github.com/libp2p/rust-libp2p/issues/2680 | https://github.com/libp2p/rust-libp2p/issues/2832 | [Kademlia client mode](#kademlia-client-mode) |
+
+Today `NetworkBehaviour` implementations like Kademlia, GossipSub or Circuit Relay v2 can not
+communicate with each other, i.e. can not make use of information known by another
+`NetworkBehaviour` implementation. First step would be to exchange standard information about remote
+peers (e.g. supported protocols).
+
+Long term we might consider a generic approach for `NetworkBehaviours` to exchange data. Though that
+would deserve its own roadmap item.
```

Comment:
I think this is an important item that we should work on because it will be a nice ergonomic improvement for our users. At the same time though, it is only just that: An ergonomic improvement. A user can integrate identify and kademlia today, they just need to write a bit of code for it.

With that in mind, I think we should remove this item from this roadmap.

Reply:
I would like to keep the paragraph here to stress that generic cross-behaviour-communication is **not** on the roadmap. Otherwise folks might confuse the scope of  this roadmap item.

Reply:
I am talking about the entire roadmap item, not just the generic approach :)